java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -524,11 +524,15 @@ private static Object[] scriptingAndFunctionsCommands(BaseTransaction<?> transac
                 "#!lua name=mylib1T \n"
                         + " redis.register_function('myfunc1T', function(keys, args) return args[1] end)";
```

Comment:
any chance you could update the function to access and use a key?  that way we would know that they're being passed successfully.

Reply:
Need to add a separate test - common transaction test considered to be keyless.
Will do

Reply:
Added in fcc4b7a8